frontend/src/metabase/actions/components/ActionFormFieldWidget/ActionFormFieldWidget.tsx

```diff
@@ -31,15 +32,36 @@ const WIDGETS: Record<InputComponentType, React.FunctionComponent<any>> = {
 
 interface FormWidgetProps {
   formField: ActionFormFieldProps;
+  hidden?: boolean;
+  onVisibilityChange?: ({ hidden }: { hidden: boolean }) => void;
 }
 
 export const ActionFormFieldWidget = forwardRef(function FormFieldWidget(
-  { formField }: FormWidgetProps,
-  ref: React.Ref<any>,
+  { formField, hidden, onVisibilityChange }: FormWidgetProps,
+  ref: Ref<any>,
 ) {
   const Widget =
     (formField.type ? WIDGETS[formField.type] : FormInputWidget) ??
     FormInputWidget;
 
-  return <Widget {...formField} nullable ref={ref} />;
+  const actions =
```

Comment:
This looks wrong from the point of composition of components. I understand that `ActionFormFieldWidget` is used in `ActionForm` and `FormFieldEditor` and we only want to get a checkbox in `FormFieldEditor`, that's why we add this `undefined` check. However it seems that we could add `actions` as a prop to `ActionFormFieldWidget` and add a checkbox in `FormFieldEditor`, and don't have the strange `undefined` check

Reply:
good point, I think I found the case when we don't need actions only after testing of the solution and it was the simplest fix 😄 I'll change it

Reply:
fixed, thanks for a good point!